.github/workflows/test-chart.yml

```diff
@@ -43,12 +43,12 @@ jobs:
       - name: helm lint
         run: |
           helm lint . \
-              --values dev-values.yaml
+              --values config/dev-values.yaml
```

Comment:
If we end up with many files, having a dedicated `dev-config` folder or similar seems reasonable, but otherwise I think its better to not create a directory for this file but instead let it reside next to values.yaml that holds the default values.

How do you think about it @tizz98 ?

Reply:
We've used a `config` folder to hold common config values for different environments, it felt a bit strange to see `dev-values.yaml` at the top level directory, but I don't have a strong opinion on this

Reply:
Ah, that sounds like a use case if you have defined a local chart that you install yourself. When we are packaging and publishing the chart, we should only provide the default values and no adjustment for any specific environment.

We do need to have some test values though, maybe `test-values.yaml` would be a better name when I think about it.